api/services/mailer/index.js

```diff
@@ -1,9 +1,11 @@
 const IORedis = require('ioredis');
-const { redis: redisConfig } = require('../../../config');
+const moment = require('moment');
+const { redis: redisConfig, app: appConfig } = require('../../../config');
 const { MailQueue } = require('../../queues');
 const Templates = require('./templates');
 
 let mailQueue;
+const { hostname } = { ...appConfig };
```

Comment:
this is a solution that @apburnes  and I paired and settled on caused by https://github.com/18F/federalist/blob/96c385bbbc58ef6ef74ed85db149cdca90cfde08/test/api/requests/main.test.js#L286 .... seems pretty unavoidable without a major overall to how we handle our config throughout the app

Reply:
I don't understand, can you explain the difference?

Reply:
the tests in this main controller test file are changing the value of the app config which are impacting other tests running and using certain app config values (in this case the mailer tests) ... and while the main controller tests attempt to revert the values for the app config after the tests conclude ... the values are still altered with the test values (ie: 'https://prod-url.com') for other test (in this case the mailer test). perhaps not reverted fast enough.

Reply:
1. As you said, this is a larger problem in that it seems the timing or order of the tests could impact whether they pass or not which is problematic, but not something we can address here.
2. What is confusing to me is why there would be a difference between the two approaches. I would think both would have the same result since I think both copy the data, ex:
```
❯ node
Welcome to Node.js v14.15.4.
Type ".help" for more information.
> const config = {
...   app: {
.....     hostname: 'host1',
.....   },
... };
> config
{ app: { hostname: 'host1' } }

> const { app: appConfig } = config;

// approach 1
> const { hostname } = { ...appConfig };

// approach 2
> const { hostname: hostname2 } = appConfig;

> console.log({ hostname, hostname2 });
{ hostname: 'host1', hostname2: 'host1' }

// set the hostname in config
> config.app.hostname = 'NEWHOST';

// result
> console.log({ hostname, hostname2 });
{ hostname: 'host1', hostname2: 'host1' }
```